src/Resources/config/services/services.xml

```diff
@@ -157,7 +157,9 @@
         />
 
         <service id="FINDOLOGIC\FinSearch\Findologic\Api\FindologicSearchService" autowire="true" />
-        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true" />
+        <service id="FINDOLOGIC\FinSearch\Findologic\Api\SortingService" autowire="true">
```

Comment:
Maybe define the service outside of the `FindologicSearchService`?

Reply:
What do you mean? This are defined as completely separate services here.

Reply:
We can leave it as it is until the service need to be injected somewhere else. It was just a suggestion.

Reply:
Can you elaborate on your suggestion? :P
I don't get the point at all

Reply:
As example see `FindologicConfigService`, defined on line 23 and injected as argument in several other lines.

Reply:
As discussed in Slack, I read it wrong, but please add empty lines between those services.